packages/record-data/addon/-private/record-data.ts

```diff
@@ -437,7 +437,7 @@ export default class RecordDataDefault implements RelationshipRecordData {
     this._destroyRelationships();
     this.reset();
     if (!this._scheduledDestroy) {
-      this._scheduledDestroy = run.backburner.schedule('destroy', this, '_cleanupOrphanedRecordDatas');
+      this._scheduledDestroy = emberBackburner.schedule('destroy', this, '_cleanupOrphanedRecordDatas');
```

Comment:
Does run complain now when you access backburner on it?

Reply:
Yes